### ISSUE TITLE
lazy font-lock when entering cm-mode

### ISSUE THREAD

**Issue by timmli** [joostkremers/criticmarkup-emacs#11]
I usually turn on `cm-mode` by hand, but this does not activate the `cm-mode` font-lock of already existing CriticMarkup elements in the buffer. The correct face is shown only with edited or new markups. How can I turn on `cm-mode` and accordingly change the face of all the CriticMarkup elements in the buffer?

**Comment by joostkremers** [comment]
Yes, I'm aware of the problem, but I've never been able to find a solution for it. I always need to edit an existing CriticMarkup element in order to make font lock work for all existing markups.

I'll try and look into this a bit more. Perhaps this time I'm more successful in finding a solution.

**Comment by timmli** [comment]
A workaround would be to disable and again enable `font-lock-mode` when entering `cm-mode`. But I'm not sure this counts as a serious solution to the issue.
